main.py

```diff
@@ -15,39 +19,133 @@ def home():
 def selfie():
     return send_file("pictures/screenshot.jpeg", mimetype="image/jpeg")
 
-
-@app.route("/recipies", methods=["GET"])
-def recipies():
-    cippie_db = sqlite3.connect("cippedb.db")
-    cursor = cippie_db.cursor()
-    recipe = cursor.execute("SELECT food_name FROM recipies")
-    return jsonify({"status": "success", "data": recipe.fetchall()})
+    
+@app.route("/recipies", methods=["GET", "POST"])
+def get_recipies():
+    if request.method == "POST":
```

Comment:
Where is the check for `"GET"`?

Reply:
I thought if it isn't POST, it must be GET since only two methods can be passed to that route